authorization/invitation/service/invitation_service.go

```diff
@@ -391,25 +396,39 @@ func (s *invitationServiceImpl) Accept(ctx context.Context, token uuid.UUID) (st
 
 			err = s.Repositories().IdentityRoleRepository().Create(ctx, ir)
 			if err != nil {
-				return resourceID, err
+				return resourceID, redirectPath, err
+			}
+		}
+
+		resource, e := s.Repositories().ResourceRepository().Load(ctx, inviteToIdentity.IdentityResourceID.String)
+		if e != nil {
+			return resourceID, redirectPath, err
+		}
+
+		if resource.ResourceType.Name == authorization.IdentityResourceTypeTeam {
+			if resource.ParentResourceID != nil {
+				redirectPath, e = s.getRelativeSpacePath(ctx, *resource.ParentResourceID)
```

Comment:
@dipak-pawar 

In future, if we allow sub-teams, or teams as a child of non-spaces:

1. check if resource type is of type space
2. if not - then redirect the user to https://openshift.io/_redirects/_acceptInvitation?status=success
3. if yes - use the space home path 

It should not be flagged as an error here as long the invitation was a valid one.

Reply:
@rohitkrai03  the above URL should be working, right?

Reply:
@sbose78 @rohitkrai03 I'd like to have a sustainable way of generating/retrieving URLs when we absolutely need it. We should avoid hard-coding URLs in the app, or even have too many URLs in configmaps/env vars.

Reply:
There can only be the following cases:

1. Use custom code for spaces : call WIT APIs, use the configured `UI URL` to generate a redirect UI URL.

2. For non-spaces, **One** Generic invitation URL for success & failure, configured as env var: `https://openshift.io/_redirects/_acceptInvitation?...`

3. Future options: 

- Modify the invitation API to allow the following info: `where should we redirect the user to, on success, and on error` example: https://che.openshift.io/workspaces/12356789

- Everytime we add a new resource type, we add custom handlers `redirectUrlOnInvitationSuccess(..)` , `redirectUrlOnInvitationFailure(..)` for each resource type.